src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -1672,6 +1674,24 @@ static string GetPackageSpecificContents(string packageName)
                          '$({NetAnalyzersSDKAssemblyVersionPropertyName})' != '' AND
                           $({NetAnalyzersNugetAssemblyVersionPropertyName}) &lt; $({NetAnalyzersSDKAssemblyVersionPropertyName})""/>
   </Target>",
+                    ResxSourceGeneratorPackageName => $@"
+  <ItemGroup>
+    <!-- Enable code generation for resource files. -->
+    <ResxCodeGenerationEmbeddedResource Include=""@(EmbeddedResource)"" Exclude=""**\*.??.resx;**\*.??-??.resx;**\*.??-????.resx"" />
+    <EmbeddedResource Update=""@(ResxCodeGenerationEmbeddedResource)"" Condition=""'%(EmbeddedResource.GenerateSource)' == ''"" GenerateSource=""true"" />
```

Comment:
How is this `GenerateSource` metadata used? I didn't bother reproducing this in my repo and everything seemed to work.

Reply:
➡️ Some analyzers add _all_ resx files to AdditionalFiles. This property allows the source generator to distinguish between ones added just for analysis, and ones added for both analysis and code generation.